frontend/src/ManageUsersPage/BulkActionsDropdown.scss

```diff
@@ -0,0 +1,20 @@
+.BulkActionsDropdown {
+  width: 22rem;
+  height: 40px;
+  overflow: hidden;
+  background: url(/static/media/chevron-down-bg.5fc1573d.svg) no-repeat right #ffffff;
+  border: 1px solid #0F0035;
+  background-size: 39px 40px;
+}
+
+.BulkActionsDropdown__select {
```

Comment:
Another stylistic thing. I like using BEM for naming and organizing my CSS classes. This helps reduce specificity and aligns pretty well with the component model of thinking about things. 

```css
.Block { 
  // component-level styles 
}
.Block__element { 
  // styles for elements of a component 
}
.Block__element--modifier { 
  // styles for an alternative version of an element. 
  // --disabled is a common use case. 
}
```
https://css-tricks.com/bem-101/

@sknep, @briaguya, Any opinions for BEM or for an alternative?

Reply:
i like --option